tests/UKHO.S100PermitService.API.FunctionalTests/Helpers/PermitServiceEndPointHelper.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace UKHO.S100PermitService.API.FunctionalTests.Helpers
+{
+    public static class PermitServiceEndPointHelper
```

Comment:
PermitServiceEndPointFactory? Move to 'Factories' namespace?

Reply:
Resolved. Thanks